### PR TITLE
Add LPAppDescription component

### PR DESCRIPTION
**Pull request by HidemichiShimura**
#### Code changes
- Created `LPAppDescription.jsx` in `src/components/pages/LP`
- Created styles `lp.module.scss`
- Added `LPAppDescription.jsx` to the entry point
- Replaced `@forward 'lp'` with `@forward lp` in `_index.scss`

#### Issues affected
-  Addresses #34 

## Additional memo

- The global color `--sub-color` has some opacity and so I applied the same color without opacity to this component background

## Screenshots and/or video

image

## Testing Plan

- Check the design responsibility with the Google Developer tools

#### Basics
- [x] Local QA is complete
- [x] No debugging `console.log()` statements
- [x] Commented-out code removed
- [x] Unused code removed (imports, functions, styles, etc - if it's not used anywhere, it's not in this PR)

### CODE REVIEW

**Comment by HidemichiShimura** [comment]
Should I delete `_lp.scss` in this PR because we are not using this anymore?

File: src/styles/pages/lp.module.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@use '../abstracts' as a;
```
Comment (Soma-dev0808) [inline src/styles/pages/lp.module.scss:1]: If you are not using modules, please remove them.

**Review by Soma-dev0808 — CHANGES_REQUESTED** [review]

File: src/styles/pages/lp.module.scss (hunk 1 above)
Comment (HidemichiShimura) [reply src/styles/pages/lp.module.scss:1]: I deleted the line importing mixin.
I also replaced the background color with the variable I added to `_varialbes.scss`

**Comment by Soma-dev0808** [comment]
LGMT!

**Review by Soma-dev0808 — APPROVED** [review]

**Review by yuya-minamide — APPROVED** [review]
lgtm

**Pull request merged**